src/Telemetry/AzureDevOpsTimeline/.config/settings.json

```diff
@@ -12,6 +12,8 @@
   "BuildBatchSize": 1000,
   "ParallelRequests": 4,
 
+  "LogScrapingTimeout": "10:00:000",
```

Comment:
Rather than having our custom format, maybe keep the default one recognized by `TimeSpan`?

`TimeSpan.Parse("10:00:000")` is 10 hours, 
`TimeSpan.Parse("00:10:00")` is 10 minutes, 

no surprises there and no need to specify a special format

Reply:
I just want to prevent everyone seeing this scratching their head on this unusual format

Reply:
1. Currently there's 10 minutes. Do you really need `ms`?
2. You can `00:00:00.300`

Reply:
We just need ms for the test, so it doesn't last a long time, I changed it to the standard format and put 00:00:00.100

Reply:
I think this still gives you 10 hours then?

Reply:
yup, forgot to change it, should be good now